docs/book/src/user/quick-start.md

```diff
@@ -539,7 +539,7 @@ clusterctl init --infrastructure docker
 In order to initialize the Equinix Metal Provider (formerly Packet) you have to expose the environment
 variable `PACKET_API_KEY`. This variable is used to authorize the infrastructure
 provider manager against the Equinix Metal API. You can retrieve your token directly
-from the [Equinix Metal Console](https://console.equinix.com/).
```

Comment:
@displague I think you added this in https://github.com/kubernetes-sigs/cluster-api/pull/5774. Do you know if there's any alternative link? This one is now giving a 403 for the MD link checker. I'd prefer to remove it if there's no easy substitution.